jboss/container/wildfly/launch/admin/added/launch/management-common.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+source $JBOSS_HOME/bin/launch/launch-common.sh
+
+# Arguments:
+# $1 - realm
+function add_management_interface_realm() {
+    local mgmt_iface_realm="${1}"
+    local mgmt_iface_replace_str
+    local mode
+    getConfigurationMode "<!-- ##MGMT_IFACE_REALM## -->" "mode"
+    if [ "x${mgmt_iface_realm}" == "x" ]; then
+      mgmt_iface_realm=ManagementRealm
+    fi
+
+    if [ "${mode}" = "xml" ]; then
+      mgmt_iface_replace_str=" security-realm=\"$mgmt_iface_realm\">"
+      sed -i "s|><!-- ##MGMT_IFACE_REALM## -->|${mgmt_iface_replace_str}|" "$CONFIG_FILE"
+    else
```

Comment:
Should be `elif [ "${mode}" = "cli" ]` (the method also returns "", in which case no replacement takes place so you can have a 'read-only' mode)